### PR TITLE
Homepage loading & footer CSS

### PR DESCRIPTION
**Pull request by emmastephenson**
# PULL REQUEST

## Summary
Two small tweaks:

* Change the home page (`localhost:3000/`) to render the upload tutorial
* Add back the footer CSS rules so it doesn't float

## Related Issue
No related issue; just some fixes

### Home page before
Screenshot 2023-11-30 at 9 00 57 AM

### Home page after (diff window sizes)
Screenshot 2023-11-30 at 9 00 08 AM
Screenshot 2023-11-30 at 9 00 19 AM

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: front-end/app/global.scss [hunk 1]
```diff
@@ -33,42 +33,46 @@
...
 
-// @media all and (min-height: 35rem){
-//   .usa-footer{
-//     position: absolute;
-//     bottom: 0;
-//     width: 100%;
-//   }
-// }
-
+@media all and (min-height: 35rem) {
```
Comment (emmastephenson) [inline front-end/app/global.scss:61]: We can blame my linter for all the other file changes here...this uncommenting is the only real change in this file!

**Comment by sarahtress** [comment]
@emmastephenson looking good! just one q:

- is the text on the bottom right of the footer splitting into 2 lines just because the screen is relatively narrow? just looks like a bit of funky behavior

everything else looks good to me though!

**Comment by emmastephenson** [comment]
> @emmastephenson looking good! just one q:
> 
> * is the text on the bottom right of the footer splitting into 2 lines just because the screen is relatively narrow? just looks like a bit of funky behavior
> 
> everything else looks good to me though!

@sarahtress yep! The screen is relatively narrow there. I tried for a while to get it to be more flexible (the 2 lines is happening sooner than I'd anticipate as the screen narrows) but no dice yet 🙃 if it's an issue I can keep trying to tweak the CSS though. Right now it's happening as soon as you go narrower than 1600px in width.

**Comment by sarahtress** [comment]
@emmastephenson ok no prob! wouldn't worry about it too much at this stage

**Review by KennethSkylight — APPROVED** [review]
lgtm!

**Pull request merged**